validator/accounts/accounts_activation.go

```diff
@@ -0,0 +1,270 @@
+					return nil
+				}
+			} else {
+				promptText := "Are you sure you want to enable %d accounts? (%s) Y/N"
+				if len(filteredPubKeys) == len(disabledPublicKeys) {
+					promptText = fmt.Sprintf("Are you sure you want to enable all accounts? Y/N (%s)", au.BrightGreen(allAccountStr))
+				} else {
+					promptText = fmt.Sprintf(promptText, len(filteredPubKeys), au.BrightGreen(allAccountStr))
+				}
+				resp, err := promptutil.ValidatePrompt(os.Stdin, promptText, promptutil.ValidateYesOrNo)
+				if err != nil {
+					return err
+				}
+				if strings.ToLower(resp) == "n" {
+					return nil
+				}
+			}
+		}
+		if err := EnableAccount(cliCtx.Context, &AccountConfig{
+			Wallet:     w,
+			Keymanager: ikeymanager,
+			PublicKeys: rawPublicKeys,
+		}); err != nil {
+			return err
+		}
+		log.WithField("publicKeys", allAccountStr).Info("Accounts enabled")
+		return nil
+	case keymanager.Derived:
+		return errors.New("cannot enable accounts for a derived keymanager")
+	default:
+		return fmt.Errorf("keymanager kind %s not supported", w.KeymanagerKind())
+	}
+}
+
+// DisableAccount disable the accounts that the user requests to be disabled from the wallet
+func DisableAccount(ctx context.Context, cfg *AccountConfig) error {
+	switch cfg.Wallet.KeymanagerKind() {
+	case keymanager.Remote:
+		return errors.New("cannot disable accounts for a remote keymanager")
+	case keymanager.Imported:
+		km, ok := cfg.Keymanager.(*imported.Keymanager)
+		if !ok {
+			return errors.New("not a imported keymanager")
+		}
+		if len(cfg.PublicKeys) == 1 {
+			log.Info("Disabling account...")
+		} else {
+			log.Info("Disabling accounts...")
+		}
+		updatedOpts := km.KeymanagerOpts()
+		existingDisabledPubKeys := make(map[[48]byte]bool, len(updatedOpts.DisabledPublicKeys))
+		for _, pk := range updatedOpts.DisabledPublicKeys {
+			existingDisabledPubKeys[bytesutil.ToBytes48(pk)] = true
+		}
+		for _, pk := range cfg.PublicKeys {
+			if _, ok := existingDisabledPubKeys[bytesutil.ToBytes48(pk)]; !ok {
+				updatedOpts.DisabledPublicKeys = append(updatedOpts.DisabledPublicKeys, pk)
+			}
+		}
+		keymanagerConfig, err := imported.MarshalOptionsFile(ctx, updatedOpts)
+		if err != nil {
+			return errors.Wrap(err, "could not marshal keymanager config file")
+		}
+		if err := cfg.Wallet.WriteKeymanagerConfigToDisk(ctx, keymanagerConfig); err != nil {
+			return errors.Wrap(err, "could not write keymanager config to disk")
+		}
+	case keymanager.Derived:
+		return errors.New("cannot disable accounts for a derived keymanager")
+	default:
+		return fmt.Errorf("keymanager kind %s not supported", cfg.Wallet.KeymanagerKind())
+	}
+	return nil
+}
+
+// EnableAccount enable the accounts that the user requests to be enabled from the wallet
+func EnableAccount(ctx context.Context, cfg *AccountConfig) error {
+	switch cfg.Wallet.KeymanagerKind() {
+	case keymanager.Remote:
+		return errors.New("cannot enable accounts for a remote keymanager")
+	case keymanager.Imported:
```

Comment:
For now, I only implemented the feature for Imported accounts.
I assume we also want to support disabling/enabling accounts for Remote/Derived but waiting for confirmation